### PR TITLE
fix: menu mobile is not transparent anymore

### LINKED ISSUE #92: fix: menu mobile is transparent
One of the users had a problem because didn't see the menu opened, so the items were above the screen, which made impossible to use the wallet.

image

### PR DESCRIPTION
**Pull request by pedroferreira1**
image

Close #92

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -299,6 +299,18 @@ svg.svg-wrapper {
...
+  }
+
+  .main-nav nav.navbar {
+    background-color: #000;
+  }
+
+  .navigation-search .version-wrapper:first-child {
+    padding: 0;
+  }
+
```
Comment (obiyankenobi) [inline src/index.scss]: Actually, I vote for removing mobile support from Desktop wallet.

Comment (msbrogli) [reply src/index.scss]: I agree. This code came from hathor-explorer and is not important here. I don't think we should remove support, but we don't have to worry about it.

Comment (obiyankenobi) [reply src/index.scss]: I'm confused.

> I agree

> I don't think we should remove support

Do you agree to remove the responsive code from the wallet? Given the bug it caused (see linked issue), I don't think we can just ignore it.

Comment (obiyankenobi) [reply src/index.scss]: Ok, let's keep mobile support

**Pull request merged**